util_network.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import asyncio
+
+
+def parse_args():
+    """
```

Comment:
bylo by možné do komentáře přidat ještě nějaký odkaz na dokumentaci? Je to pro mě hatmatilka :-)